docs/COMMANDS.md

```
@@ -0,0 +1,1317 @@
+tuti init
+
+# Initialize with Laravel stack directly
+tuti init myproject --stack=laravel
+
+# Force reinitialize
+tuti init --force
+```
+
+**What It Creates**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)